src/binding/python/openpmd_api/pipe/__main__.py

```diff
@@ -47,7 +47,8 @@ def parse_args(program_name):
 
 Examples:
     {0} --infile simData.h5 simData_%T.bp
```

Comment:
Is this missing here as well?
```suggestion
    {0} --infile simData.h5 --outfile simData_%T.bp
```

Reply:
You're absolutely right. I guess this was a copy-paste error then